src/BalconyBotanica/Core/DomainObjects/PlantFunction.cs

```diff
@@ -0,0 +1,10 @@
+namespace BalconyBotanica.Core.DomainObjects
+{
+    public enum PlantFunction
+    {
+        EATABLE,
```

Comment:
Not related to programming but the correct English word should be:

```suggestion
        EDIBLE,
```